tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/language_script.tsv

```diff
@@ -172,6 +172,8 @@ cs	Czech	primary	Latn	Latin
 csb	Kashubian	secondary	Latn	Latin
 csw	Swampy Cree	primary	Cans	Unified Canadian Aboriginal Syllabics
```

Comment:
csw and probably others may be secondary latin (they call it SRO - standard roman orthography)